src/controllers/dlgprefcontroller.cpp

```diff
@@ -311,15 +312,21 @@ void DlgPrefController::enumeratePresets(const QString& selectedPresetPath) {
 
     // qDebug() << "Enumerating presets for controller" << m_pController->getName();
 
+    // Check the text color of the palette for whether to use dark or light icons
+    QDir iconsPath = QString(":/images/preferences/dark/");
```

Comment:
Use a default constructed QDir and the setPath() in both symmetric if/else arms.

Reply:
Okido, makes sense. If that's all you prefer to be changed I'd ammend those changes to 1b165bb752ab6cff3b986a8da169126d2e0e8527 and force-push

Reply:
just drop a line here (no reaction emoji, I probably don't notice those)

Reply:
_ping_ @uklotzde